### DISCUSSION TITLE
mkdocs-monorepo-plugin feature parity?

### DISCUSSION THREAD

**Discussion by jgarte** [useblocks/sphinx-collections#9] [Q&A]
Hi,

How does `sphinx-collections` compare to [mkdocs-monorepo-plugin](https://github.com/backstage/mkdocs-monorepo-plugin) or [mkdocs-multirepo](https://github.com/wilhelmer/mkdocs-multirepo)?

Is there any feature parity between the two?

**Comment by danwos** [comment]
I haven't used one of them. 

And I think [mkdocs-monorepo-plugin](https://github.com/backstage/mkdocs-monorepo-plugin) is the one, which is similar to Sphinx-Collections, as it merges multiple documentation projects into one. Which is one of the main motivations of Sphinx-Collection.

For me [mkdocs-multirepo](https://github.com/wilhelmer/mkdocs-multirepo) creates a nice landing/starting page and links from it to multiple, standalone documentation projects. That is something Sphinx-Collection does not have, but I think could be realized with a self-developed driver for the starting-page and some config to collect all needed documentation folders.

**Reply by jgarte** [reply]
Hi, thanks for the reply.

What does it mean for a driver to be self-driven?

**Reply by danwos** [reply]
Sorry, the word should have been "self-developed". I changed it already in my answer.

**Comment by jgarte** [comment]
>  but I think could be realized with a self-developed driver for the starting-page and some config to collect all needed documentation folders.

@danwos 

Do you think that this feature could be implemented as an extension to the `JinjaDriver` or by some other means?
